phpunit.xml.dist

```diff
@@ -39,7 +42,28 @@
 			<directory>./tests/system/Database</directory>
 		</testsuite>
 	</testsuites>
-	<logging/>
+
+	<listeners>
+		<listener class="JohnKary\PHPUnit\Listener\SpeedTrapListener">
+			<arguments>
+				<array>
+					<!-- Number of milliseconds when a test is considered "slow" (Default: 500ms) -->
+					<element key="slowThreshold">
+						<integer>500</integer>
```

Comment:
We should set our minimum slowness threshold here.

Reply:
I think starting at 500 (the default) makes sense. We can inline different thresholds or specific tests to ignore once we identify ones that "can't" be fixed.

This might be way beyond the scope of this PR but... I would love annotations from this.😅 Could you look into if anyone has found a solution for that yet?

Reply:
Annotations like this?
```php
/** @slowThreshold 1000 */
public function testAnythingHere(): void
{
    // ...
}
```

Reply:
No, some Action services will actually annotate the file under Review. PHPStan does it. I can't find an example right now but I can create one real quick.

Reply:
See here for how the Action result gets overlaid with its context: https://github.com/codeigniter4/CodeIgniter4/pull/4451/files